src/service/programService.js

```diff
@@ -219,12 +219,85 @@ async function deleteProgram(req, response) {
   });
 }
 
+function getProgramCountsByOrg(req, response) {
+  var data = req.body
+  var rspObj = req.rspObj
+  rspObj.apiId = 'api.program.counts';
+  rspObj.apiVersion = '1.0';
+  rspObj.msgid =  uuid();
+  rspObj.result = {};
+  rspObj.errCode = programMessages.PROGRAMCOUNTS_BYORG.PROGRAMCOUNTS_FETCH.FAILED_CODE
+  rspObj.errMsg = programMessages.PROGRAMCOUNTS_BYORG.PROGRAMCOUNTS_FETCH.FAILED_MESSAGE
+  rspObj.responseCode = '';
+
+  const findQuery = (data.request && data.request.filters) ? data.request.filters : {}
+  const facets = ["rootorg_id"];
```

Comment:
Yeah. We could change to getting it from request instead of hardcoding it. @pallakartheekreddy can take a call on that.